wai-aria/role/contextual-roles.html

```diff
@@ -0,0 +1,102 @@
+<!doctype html>
+<html>
+<head>
+  <title>Contextual Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<p>Verifies Required Context Roles from <a href="https://www.w3.org/TR/wai-aria-1.2/#scope">5.2.7 Required Context Role</a></p>
+<p>Also verifies Required Owned Roles from <a href="https://www.w3.org/TR/wai-aria-1.2/#mustContain">5.2.6 Required Owned Role</a></p>
+
+<!-- Notes for "required context" testing:
+
+- See Computed Role for more details on role computation when an element lacks required context: 
+https://w3c.github.io/core-aam/#roleMappingComputedRole 
+
+- See ARIA 1.2 spec for full listing of "Required Context Role" for each element:
+https://www.w3.org/TR/wai-aria-1.2/
+
+Identified roles with "Required Context" excluding abstract roles (e.g., child -> parent):
+
+- caption -> {figure, grid, table, treegrid}
+- cell -> row
+- columnheader -> row
+- gridcell -> row
```

Comment:
At least some are in ./table-roles.html

Reply:
It looks like table-roles doesn't exist? Happy to create 👍🏾

Reply:
Right... there's a `/html-aam/table-roles.html`, but I agree that this should be a new file in `/wai-aria`

Please add it, and include comment pointers to/from the related files:
- `/html-aam/table-roles.html`
- `/wai-aria/grid-roles.html`

such as:

```html
<!-- grid roles tested in ./grid-roles.html -->
```

and vice versa pointers back to your new file from the others. Thanks!